dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -127,3 +127,16 @@ startsecs=10
 dependent_startup=true
 dependent_startup_wait_for=start:exited
 {% endif %}
+
+{% if not skip_pcied %}
```

Comment:
I still question if we need to provide the ability to skip the PCIe check. I think we may want to require it from all platforms. I guess we can leave this here temporarily until all vendors implement it, and then remove it.

Reply:
@jleveque  all other daemon has condition like this and to skip this daemon, each platform vendor specifically add this skip condition in there pmon_daemon_control.json. So do you want to remove all the conditions in this configuration file?

Reply:
No, don't remove them all. Some are currently used because vendors haven't yet implemented some APIs, like the thermal API, so they disable that daemon temporarily. Also, we are disabling platform daemons for virtual switches, because there is no physical platform on a virtual device. Eventually, I would like to see all daemons running on all physical platforms, and then maybe we can find a different disabling mechanism for virtual platforms (maybe disable the PMon container entirely?).

However, for now, if pcied will output errors on platforms which haven't implemented it, we can leave this.

Reply:
@jleveque It is how it works right now. pcied exits with printing error message that it can't find a configuration file for the platform. This skip option is not necessary but I just added to make the pcied configuration same as other daemon configuration since it's no harm to have this condition.

Reply:
Having the option makes it appear as though the daemon is optional, but we would like the daemon to be mandatory. I would eventually like to remove these options and create a different mechanism to disable the daemons on virtual switch platforms only, once most vendors have implemented all of the APIs. Until then, we can leave this.